lottie/src/main/java/com/airbnb/lottie/LottieFeatureFlags.java

```diff
@@ -0,0 +1,62 @@
+package com.airbnb.lottie;
+
+import android.annotation.SuppressLint;
+import android.os.Build;
+
+import com.airbnb.lottie.utils.Logger;
+
+import java.util.HashMap;
+import java.util.Objects;
+
+public class LottieFeatureFlags {
+  public enum FeatureFlag {
+    /**
+     * Merge paths currently don't work if the the operand shape is entirely contained within the
+     * first shape. If you need to cut out one shape from another shape, use an even-odd fill type
+     * instead of using merge paths.
+     */
+    MergePath_19(Build.VERSION_CODES.KITKAT);
+
+    public final int minRequiredSdkVersion;
+
+    FeatureFlag(int minRequiredSdkVersion) {
+      this.minRequiredSdkVersion = minRequiredSdkVersion;
+    }
+  }
+
+  private HashMap<FeatureFlag, Boolean> flagValues = new HashMap<>();
+
+  private static LottieFeatureFlags INSTANCE;
+
+  public static LottieFeatureFlags getInstance() {
+    if (INSTANCE != null) {
+      return INSTANCE;
+    } else {
+      INSTANCE = new LottieFeatureFlags();
+      return INSTANCE;
+    }
+  }
+
+  private LottieFeatureFlags() {
+    for (FeatureFlag f : FeatureFlag.values()) {
+      flagValues.put(f, Boolean.FALSE);
+    }
+  }
+
+  @SuppressLint("DefaultLocale")
+  public void enableFlag(FeatureFlag flag, boolean enable) {
+    if (Objects.equals(flagValues.get(flag), enable)) {
```

Comment:
Does this avoid autoboxing of a nullable boolean or does it always autobox boolean into Boolean because the equals param is Object?